modules/priceFloors.js

```diff
@@ -568,8 +568,9 @@ export function handleSetFloorsConfig(config) {
   _floorsConfig = utils.pick(config, [
     'enabled', enabled => enabled !== false, // defaults to true
     'auctionDelay', auctionDelay => auctionDelay || 0,
+    'floorProvider', floorProvider => String(floorProvider) || '',
     'endpoint', endpoint => endpoint || {},
-    'skipRate', () => !isNaN(utils.deepAccess(config, 'data.skipRate')) ? config.data.skipRate : config.skipRate || 0,
+    'skipRate', () => !isNaN(utils.deepAccess(config, 'data.skipRate')) ? config.data.skipRate : config.skipRate || undefined,
```

Comment:
We want to actually leave in the default of `0`. The goal with the Issue is to give analytics adapters the ability to infer what is happening. The adding of `noData` to the `location` enum succeeds in doing this.

We do not want `skipRate` to be a required field, so let's leave it at zero.